src-js/optimizely.js

```diff
@@ -40,6 +40,9 @@ class OptimizelyClient {
         return reject({error: "Missing required options"});
       }
 
+      // defines additional attributes we will want to send to optimizely to qualify/disqualify a user
+      const attributes = options.attributes ?? {};
+
```

Comment:
Ah I see, going off of my previous comment, is the attribute that you included in `getVariationName` another check that the user needs to satisfy to be able to qualify for an experiment?